maudtools/generating_sbml.py

```diff
@@ -3,8 +3,21 @@
 This function takes in an InferenceData, MaudInput, chain, draw and experiment
```

Comment:
I am getting the following error (`maudtools generate-sbml output_dir`):

```
Traceback (most recent call last):
  File "/home/georg/.virtualenvs/maud-local/bin/maudtools", line 8, in <module>
    sys.exit(cli())
  File "/home/georg/.virtualenvs/maud-local/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/georg/.virtualenvs/maud-local/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/georg/.virtualenvs/maud-local/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/georg/.virtualenvs/maud-local/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/georg/.virtualenvs/maud-local/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/georg/.virtualenvs/maud-local/lib/python3.9/site-packages/maudtools/cli.py", line 112, in generate_sbml_command
    experiment.id for experiment in mi.measurements.experiments
AttributeError: 'MaudInput' object has no attribute 'measurements'
```

Reply:
Thanks, the sbml now works on my computer

Reply:
Only balanced metabolites are reported as `Species` when I run it, generating in the end empty reactions (empty meaning without metabolites, although they have kinetic laws). Is that intended?

Reply:
I think this is intended - the unbalanced metabolites appear in Maud's Stan model  as a parameter array called `conc_unbalanced` so I thought they should be represented in sbml in the same way as other parameters.

Reply:
Okay, I see. 

Another way to do it is to add unbalanced metabolites as any other [`Species`](https://docs.rs/rust_sbml/latest/rust_sbml/struct.Species.html) with the attribute `boundary_condition` set to "true". This way the model can be parsed and inspected in a normal way, but it may be ambiguous if the parsing is not careful (which is a concern, I believe).

If you think it is fine as it is I can approve the PR, I don't have any more comments.

Reply:
I'd quite like to get this change merged soon but I've made an issue to look into the best way to do this separately.